### PR TITLE
Verbose lnx error output, fix formatting and update deps

### PR DESCRIPTION
**Pull request by atuchin-m**
If the container returns error then explicitly write about it in the response body.
This helps to diff go-translate issues from lnx container issues.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -14,14 +14,14 @@ jobs:
...
+      - uses: actions/checkout@v4
 
       - uses: actions/setup-go@cdcb36043654635271a94b9a6d1392de5bb323a7 # v5
         with:
-          go-version: 1.17
+          go-version: 1.22
 
       - name: lint
-        uses: golangci/golangci-lint-action@v3
+        uses: golangci/golangci-lint-action@v6
```
[the remark this answers is not in the document]
Comment (atuchin-m) [reply .github/workflows/ci.yml:24]: This is supposed to be pinned right after merging this PR (that updates the actions)

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ jobs:
     name: ci
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@44c2b7a8a4ea60a981eaca3cf939b5f4305c123b # v4
```
Comment (thypon) [inline .github/workflows/ci.yml:17]: here it's better to use the full version `4.1.7`

Comment (thypon) [reply .github/workflows/ci.yml:17]:
```suggestion
      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
```

**Review by thypon — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -14,14 +14,14 @@ jobs:
...
 
       - uses: actions/setup-go@cdcb36043654635271a94b9a6d1392de5bb323a7 # v5
         with:
-          go-version: 1.17
+          go-version: 1.22
 
       - name: lint
-        uses: golangci/golangci-lint-action@v3
+        uses: golangci/golangci-lint-action@v6
```
Comment (thypon) [inline .github/workflows/ci.yml:24]:
```suggestion
        uses: golangci/golangci-lint-action@a4f60bb28d35aeee14e6880718e0c85ff1882e64 # v6.0.1
```

**Review by thypon — DISMISSED** [review]

**Review by linhkikuchi — APPROVED** [review]

**Pull request merged**